### PR TITLE
Add more documentation on Diffy, Sampler usage

### PR DESCRIPTION
**Pull request by idreeskhan**
#53 #34

### CODE REVIEW

File: ratatool-diffy/diffy.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+ * `global` - Global counts of Diff types (`SAME`, `DIFFERENT`, `MISSING_LHS`, `MISSING_RHS`) seen in the entire dataset
+ * `fields` - Field level statistics including, but not limited to, the number of records with different values per field, min, max, standard deviation 
+ * `keys` - All unique keys found in the two datasets and their Diff types by key
+
+For full details see [BigDiffy.scala](https://github.com/spotify/ratatool/blob/master/ratatool-diffy/src/main/scala/com/spotify/ratatool/diffy/BigDiffy.scala)
+
+## Usage
+
+```
```
Comment (jbigred1) [inline ratatool-diffy/diffy.md]: do we want to discuss the output and how to read it as well

File: ratatool-sampling/sampler.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+Sampler
+=======
+
+Diffy contains record sampling classes for Avro, BigQuery, Parquet.
+
+# BigSampler
+
+BigSampler will run a [Scio](https://github.com/spotify/scio) pipeline sampling either Avro or BigQuery data.
```
Comment (jbigred1) [inline ratatool-sampling/sampler.md]: can we note that it supports `gs://` paths as well

File: ratatool-sampling/src/main/scala/com/spotify/ratatool/samplers/BigSampler.scala [hunk 3]
```diff
@@ -96,15 +113,17 @@ object BigSampler {
...
-    val output = args("output")
-    val fields = args.list("fields")
-    val seed = args.optional("seed")
+    val (samplePct, input, output, fields, seed) = try {
+      val pct = args("sample").toFloat
+      require(pct > 0.0F && pct <= 1.0F)
+      (pct, args("input"), args("output"), args.list("fields"),
+        args.optional("seed"))
+    } catch {
+      case e: Throwable =>
```
Comment (jbigred1) [inline ratatool-sampling/src/main/scala/com/spotify/ratatool/samplers/BigSampler.scala]: do we want to catch only `NonFatal` here instead of everything

http://www.scala-lang.org/api/current/scala/util/control/NonFatal$.html

File: ratatool-diffy/diffy.md (hunk 1 above)
Comment (idreeskhan) [reply ratatool-diffy/diffy.md]: It's discussed a bit in the earlier section. Do you think it needs to be more detailed?

Comment (jbigred1) [reply ratatool-diffy/diffy.md]: Yeah maybe give an example of how to read the output since I know we mentioned it's hard to parse potentially

File: ratatool-sampling/sampler.md (hunk 2 above)
Comment (idreeskhan) [reply ratatool-sampling/sampler.md]: Added
